src/core/qgssnappingconfig.h

```diff
@@ -168,18 +182,55 @@ class CORE_EXPORT QgsSnappingConfig
          */
         void setTypeFlag( QgsSnappingConfig::SnappingTypeFlag type );
 
-        //! Returns the tolerance
+        /**
+         * !
+         * Returns the tolerance
+         * \since QGIS 3.12
```

Comment:
I don't think this method and the following were added in 3.12
https://github.com/qgis/QGIS/blame/master/src/core/qgssnappingconfig.h#L172

Reply:
Yes I think I went too fast with some doc modification.